aleth-bootnode/main.cpp

```diff
@@ -132,9 +124,21 @@ int main(int argc, char** argv)
 
     Host h(c_programName, netPrefs, &netData);
     h.start();
+    if (!h.haveNetwork())
+        return -1;
 
     cout << "Node ID: " << h.enode() << endl;
 
+    if (!noBootstrap)
+    {
+        for (auto const& bn : Host::pocHosts())
```

Comment:
Can you rename `pocHosts` to something else? `defaultBootstrapNodes`? Also, I don't think it should be a method of `Host`.

Reply:
Agreed, I never liked that it was a part of Host...I think I'll move it to Network.h